hack/update-gofmt.sh

```diff
@@ -34,6 +34,7 @@ find_files() {
         -o -wholename './_output' \
         -o -wholename './_gopath' \
         -o -wholename './release' \
+        -o -wholename '*/staging/*' \
```

Comment:
sorry to be a PITA - dowe have any evidence for a `staging` dir in anything but the root?  I.e. why not './staging/*'